res/compile.py

```diff
@@ -0,0 +1,193 @@
+import os
+import subprocess
+import re
+import binascii
+import struct
+from multiprocessing.pool import ThreadPool
+
+# Set the path to the fxc compiler
+fxc = 'C:\\Program Files (x86)\\Microsoft DirectX SDK (June 2010)\\Utilities\\bin\\x86\\fxc.exe'
```

Comment:
The reason was simply that I didn't take the time to look up how to query system variables. I've updated the script.